Model/Transaction/Refund.php

```diff
@@ -77,9 +77,26 @@ public function build(
         );
 
         if (isset($this->request['line_items'])) {
-            foreach ($this->request['line_items'] as $lineItem) {
+            $adjustmentFee = $creditmemo->getAdjustmentNegative();
```

Comment:
Any thoughts on moving this new functionality to its own method?

Reply:
In a future PR I'd love to refactor a lot of this, once we have more tests in place. 😀